src/components/CourseCard.js

```diff
@@ -47,23 +38,18 @@ function CourseCard (props) {
               {props.view==="student" ? 
               <Link to={`/studentSession`}>
                 <Button className="joinBtn">
-                  Join Session
+                  Join Session - Not Functional
                 </Button>
               </Link> : 
               <div></div>}
 
               {/*View Course Button*/}
-              {props.view==="student" ? 
-              <Link to={`/singleCourse`}>
-                <Button className="hideBtn" onClick={() => {saveCourseClicked(props.currentCourse)}}>
-                  View Course
-                </Button>
-              </Link> : 
-              <Link to={`/singleCourse`}>
+               
+              <Link to={`/${props.course.id}`}>
                 <Button className="hideBtn" onClick={() => {saveCourseClicked(props.currentCourse)}}>
```

Comment:
Does this still go to a saveCourseClicked Function? I can't seem to find where this onclick is supposed to go now that the function is removed